use-cases.md

```diff
@@ -86,44 +86,44 @@ def top_level_function():
 
-  1. Attempt to join an existing trace given a Span that's been propagated alongside the incoming request (in case the trace has already been started by the client), or create a new trace if no such propagated Span could be found.
+  1. Attempt to extract a SpanContext that's been propagated alongside the incoming request (in case the trace has already been started by the client), or start a new trace if no such propagated SpanContext could be found.
   1. Store the newly created Span in some _request context_ that is propagated throughout the application, either by application code, or by the RPC framework.
   1. Finally, close the Span using `span.finish()` when the server has finished processing the request.
 
-#### Joining to a Trace from an Incoming Request
+#### Extracting a SpanContext from an Incoming Request
 
-Let's assume that we have an HTTP server, and the Span is propagated from the client via HTTP headers, accessible via `request.headers`:
+Let's assume that we have an HTTP server, and the SpanContext is propagated from the client via HTTP headers, accessible via `request.headers`:
 
 {% highlight python %}
-span = tracer.join(
-    operation_name=operation,
+extracted_context = tracer.extract(
     format=opentracing.HTTP_HEADER_FORMAT,
     carrier=request.headers
 )
 {% endhighlight %}
 
-Here we set both arguments of the decoding method to the `headers` map. The Tracer object knows which headers it needs to read in order to reconstruct the tracer state and any Baggage.
+Here we use the `headers` map as the carrier. The Tracer object knows which headers it needs to read in order to reconstruct the tracer state and any Baggage.
 
-The `operation` above refers to the name the server wants to give to the Span. For example, if the HTTP request was a POST against `/save_user/123`, the operation name can be set to `post:/save_user/`. OpenTracing API does not dictate how applications name the spans.
+#### Continuing or Starting a Trace from an Incoming Request
 
-#### Joining to or Starting a Trace from an Incoming Request
-
-The `span` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
+The `extracted_context` object above can be `None` if the Tracer did not find relevant headers in the incoming request: presumably because the client did not send them. In this case the server needs to start a brand new trace.
 
 {% highlight python %}
-span = tracer.join(
-    operation_name=operation,
+extracted_context = tracer.extract(
     format=opentracing.HTTP_HEADER_FORMAT,
     carrier=request.headers
 )
-if span is None:
+if extracted_context is None:
     span = tracer.start_span(operation_name=operation)
+else:
+    span = tracer.start_span(operation_name=operation, child_of=extracted_context)
```

Comment:
re `RPCParent`: I see what you're getting at, but I'd like to revisit this once we've hashed through the implications of all of this in `opentracing-python` if that's ok with you. Otherwise we'll just end up changing this example twice.